app/models/target.rb

```diff
@@ -0,0 +1,31 @@
+# == Schema Information
+#
+# Table name: targets
+#
+#  id         :bigint           not null, primary key
+#  location   :point
+#  radius     :integer          not null
+#  title      :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  topic_id   :bigint           not null
+#  user_id    :bigint           not null
+#
+# Indexes
+#
+#  index_targets_on_topic_id  (topic_id)
+#  index_targets_on_user_id   (user_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (topic_id => topics.id)
+#  fk_rails_...  (user_id => users.id)
+#
+class Target < ApplicationRecord
+  belongs_to :topic, dependent: :destroy
+  belongs_to :user, dependent: :destroy
```

Comment:
Do you want to destroy Topics and Users when a target is destroyed? I think that the dependent destroy should be on the other side of the relationships